src/Akka.Persistence.Sql/persistence.conf

```diff
@@ -16,12 +16,12 @@
       # If true, journal_metadata is created and used for deletes
       # and max sequence number queries.
       # note that there is a performance penalty for using this.
-      delete-compatibility-mode = true
+      delete-compatibility-mode = false
```

Comment:
Change `delete-compatibility-mode` to false, we want to push the use of the new tag table for CQRS reasons

Reply:
LGTM